### ISSUE TITLE
Allow publishing OOPS! reports too

### ISSUE THREAD

**Issue by dgarijo** [OnToology/OnToology#381]
When documentation is published, it would be nice if you could configure whether to publish OOPS! reports as well.

Raised by @micheldumontier

**Comment by dgarijo** [comment]
Isn't this already in OnToology, @ahmad88me ?

**Comment by ahmad88me** [comment]
@dgarijo You are right, it is published, but there is no direct link to access the report from OnToology (the user has to use a long url which is not user friendly).

**Comment by mariapoveda** [comment]
I like this feature but it should be configurable in the config file I guess. Also now we have the markdown report published in the github as alternative report.
